src/api/PostManage2fa.ts

```diff
@@ -0,0 +1,25 @@
+import { AxiosRequestConfig } from 'axios';
+import axiosInstance from '@redhat-cloud-services/frontend-components-utilities/interceptors/interceptors';
+import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
+
+export interface PostManageOrg2faRequest {
+  enable_org_2fa: boolean;
+}
+
+export const postManageOrg2fa = async (request: PostManageOrg2faRequest) => {
+  const chrome = useChrome()
```

Comment:
We can't call `useChrome` in here - check how we pass the options in the command processor and maybe you could pass the `auth` object.